contents/blog/why-redesign.mdx

```diff
@@ -0,0 +1,171 @@
+---
+title: Why are we redesigning PostHog?
+date: 2023-11-27
+author: ["cory-watilo"]
+rootpage: /blog
+featuredImage: ../images/blog/why-redesign/preview-3000.jpg
+featuredImageType: full
+category: PostHog news
+tags:
+  - Product updates
+---
+
+A year ago, [James](/james) and I sat down in a coffee shop in Austin, Texas and had a discussion about what the future of PostHog could look like.
+
+Our current UI has served its purpose, but as we grow as a company, the possibilities of what can be achieved in PostHog have grown – as have our ambitions.
+
+We’re launching a reskin this month as the first step in a series of many milestones around the experience of using PostHog. I’d like to share some of the reasoning behind our changes, and the longer-term vision we’re aiming to realize in the future.
+
+## Why we're making changes
+
+Sometimes it feels like designers change things for the sake of change. I promise, this is not one of those times. Here's some of the thinking that's gone into it:
+
+### 1. PostHog isn’t product analytics - it’s a platform
+
+As of this writing, we’re at five products with three more in beta. Interoperability will be a key differentiator for PostHog.
+
+Underlying every query is our new [HogQL querying engine](/docs/hogql). This means it’s now possible to display a graph of users who match a set of criteria, then pivot to watching a playlist of session recordings, then toss those users into a cohort, enable a feature flag, and send them an in-app survey – all stemming from the same underlying query.
+
+This means PostHog needs to be flexible, visually agnostic, and adaptable to pivot to whatever type of querying is being done.
```

Comment:
Q: I was planning on writing another piece about Notebooks launching. Do we want that to launch at the same time? Should it be linked/mentioned here, rather than below?